3-ecommerce-db/projects/cyf-ecommerce-api/server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "server-package",
+  "version": "1.0.0",
+  "description": "server",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "dev": "set NODE_ENV=local && nodemon server.js",
+    "test": "echo 'Error: no test specified' && exit 1"
+  },
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv-flow": "^4.0.0",
+    "express": "^4.18.2",
+    "pg": "^8.11.3"
+  }

```

Comment:
you have this node script configured: 
`"dev": "set NODE_ENV=local && nodemon server.js",`
this means you want to run 2 commands in one line:
```sh
set NODE_ENV=local
nodemon server.js
```
but in script, you stick them together with ` &&` and that allows to execute them one by one in just one line.

so, having said that, you see the second part of that thingy:
```sh
nodemon server.js
```
here `nodemon` is a program, and `server.js` is the file you want to run in that program.
but problem is that `nodemon` is not something that you have in your computer by default, 
you should get it installed somehow.
in this case you can get it installed as a Node Dependency, 
normally, under the `devDependencies` section in your json file:
```json
"devDependencies": {
  "nodemon": "^ .... whatever version you wanna put here"
}
```

if you don;t have that, then you can't run start server with `npm run dev` - it'll fail.